var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -61,6 +61,16 @@ class Dyninst(CMakePackage):
     patch('v9.3.2-auto.patch', when='@9.3.2 %gcc@:4.7.99')
     patch('tribool.patch', when='@9.3.0:10.0.0 ^boost@1.69:')
 
+    # We currently only build with gcc
+    conflicts('%clang')
```

Comment:
There is also a `%apple-clang` compiler now

Reply:
Is that case not handled by #18219?

Reply:
Technically it is since you can't build `%apple-clang` on a non-macOS system.

Reply:
Ok. Maybe we should merge that one first. Then I'll rebase this, if necessary.

Reply:
I can't merge mine until someone reviews and approves it

Reply:
Honestly, you could just add that conflict to your PR and I can close my PR.

Reply:
Done.